roles/openshift_control_plane/tasks/static_shim.yml

```diff
@@ -10,10 +10,10 @@
   - "scripts/{{ l_runtime }}/master-logs"
   - "scripts/{{ l_runtime }}/master-restart"
   vars:
-    l_runtime: "{{ 'crio' if openshift_use_crio | default(False) else 'docker' }}"
+    l_runtime: "{{ 'crio' if openshift_use_crio | default(False) | bool else 'docker' }}"
```

Comment:
This looks like it should be
```yaml
"{{ 'crio' if (openshift_use_crio | bool) | default(False) else 'docker' }}"
```

Otherwise, 'crio' if openshift_use_crio will resolve to true for any string other than "False"